.changeset/curvy-boxes-poke.md

```diff
@@ -3,6 +3,10 @@
 '@spectrum-web-components/styles': patch
 ---
 
+Adjust S1/Express static outline variant content color (from transparent-black/white to solid black/white) to fix unintentional regression.
+
+From [@spectrum-css/button v14.1.6](https://www.npmjs.com/package/@spectrum-css/button/v/14.1.6): [#3665](https://github.com/adobe/spectrum-css/pull/3665) Thanks [@rise-erpelding](https://github.com/rise-erpelding)!
+
```

Comment:
It looks like this is now conflicting with the main branch--this was added because it was meant to be in the other button PR (#5363) but overlooked there, so we were hoping to get this additional note on style updates added to that changeset, since it does not apply to this PR/changeset but should still be included.

Now that we've released, should this maybe go into button's `CHANGELOG.md` instead?